test/functional/subscription_mailer_test.rb

```diff
@@ -0,0 +1,12 @@
+require 'test_helper'
+
+class SubscriptionMailerTest < ActionMailer::TestCase
+  test "notify_tag_added" do
```

Comment:
Awesome. Great great great. Love that this test is already here. Let's do it as part of the separate mail template PR, ok?